File: C_Programs/SalaryBreakupPrintInFormat.cpp
//wap to accept empno empname and basic_salary then calculate hra (15% of basic salary) , da (25% of basic) , ta (55% of basic) , total salary(basic_Salary + hra + da+ ta)

#include<stdio.h>
#include<conio.h>

int main()
{
	int cnt,empno,basicsal;
	char empname[20];
	float hra,da,ta,totalsal;
	
	printf("Enter the Employment No : ");
	scanf("%d",&empno);
	printf("\nEnter the Employee Name : ");
	scanf("%s",&empname);
	printf("\nEnter the basic salary : ");
	scanf("%d",&basicsal);
	
	hra=0.15f*basicsal;
	da=0.25f*basicsal;
	ta=0.55F*basicsal;
	totalsal=basicsal+hra+ta+da;
	
	printf("\n------------------------------------------------------------------------------------------------------------------------");
	printf("Emp.No.\t Emp.Name\t\t Basic Salary\t\t HRA\t\t DA\t\t TA\t\t Total Salary");
	printf("\n%d\t %s\t\t %d\t\t \t %.2f \t %.2f \t %.2f \t %.2f",empno,empname,basicsal,hra,da,ta,totalsal);
	printf("\n------------------------------------------------------------------------------------------------------------------------");
	return 0;
}

File: Core_Java_Programs/SalaryBreakupPrintInFormat.java
//wap to accept empno empname and basic_salary then calculate hra (15% of basic salary) , da (25% of basic) , ta (55% of basic) , total salary(basic_Salary + hra + da+ ta)

import java.util.Scanner;

public class SalaryBreakupPrintInFormat {
    public static void main(String[] args)
    {
        int cnt,empno,basicsal;
        String empname;
        float hra,da,ta,totalsal;
        Scanner sc=new Scanner(System.in);

        System.out.println("Enter the Employment No : ");
        empno=sc.nextInt();
        System.out.println("Enter the Employee Name : ");
        empname=sc.next();
        System.out.println("Enter the basic salary : ");
        basicsal=sc.nextInt();

        hra=0.15f*basicsal;
        da=0.25f*basicsal;
        ta=0.55F*basicsal;
        totalsal=basicsal+hra+ta+da;

        System.out.println("------------------------------------------------------------------------------------------------------------------------");
        System.out.println("Emp.No.\tEmp.Name\t\t Basic Salary\t\t HRA\t\t DA\t\t TA\t\t Total Salary");
        System.out.println(empno+"\t\t\t" +empname+"\t\t\t"+ basicsal+"\t\t \t"+hra+" \t\t"+ da+" \t "+ta+" \t"+totalsal);
        System.out.println("------------------------------------------------------------------------------------------------------------------------");
    }
}
